backend/sql/postgresql/schema/20231108001411_invites.up.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE IF NOT EXISTS neosync_api.invites (
+	id uuid NOT NULL DEFAULT gen_random_uuid(),
+  account_id uuid NOT NULL,
+  sender_user_id uuid NOT NULL,
+  email varchar NOT NULL,
+  token varchar NOT NULL DEFAULT gen_random_uuid(),
+  accepted boolean DEFAULT false,
+	created_at timestamp NOT NULL DEFAULT now(),
+	updated_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  expires_at timestamp NOT NULL,
+	CONSTRAINT invites_pkey PRIMARY KEY (id),
+  CONSTRAINT fk_invites_accounts_id FOREIGN KEY (account_id) REFERENCES neosync_api.accounts(id) ON DELETE CASCADE,
+  CONSTRAINT fk_invites_users_id FOREIGN KEY (id) REFERENCES neosync_api.users(id) ON DELETE CASCADE,
+  CONSTRAINT invites_email_account_id UNIQUE (email, account_id)
```

Comment:
I don't know if we want this constraint. If an email gives invited but the invitation expires, they should be able to receive another invite without having to delete the previous one.